shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/jaxb/cases/domain/segment/impl/table/ExpectedJoinTable.java

```diff
@@ -20,23 +20,29 @@
 import lombok.Getter;
 import lombok.Setter;
 import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.segment.AbstractExpectedDelimiterSQLSegment;
-import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.segment.impl.ExpectedJoinSpecification;
+import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.segment.impl.column.ExpectedColumn;
+import org.apache.shardingsphere.test.sql.parser.parameterized.jaxb.cases.domain.segment.impl.expr.ExpectedExpression;
 
 import javax.xml.bind.annotation.XmlElement;
+import java.util.LinkedList;
+import java.util.List;
 
 /**
  * Expected JoinTable.
  */
 @Getter
 @Setter
 public final class ExpectedJoinTable extends AbstractExpectedDelimiterSQLSegment {
-    
+
     @XmlElement(name = "left")
     private ExpectedTable left;
 
     @XmlElement(name = "right")
     private ExpectedTable right;
-    
-    @XmlElement
-    private ExpectedJoinSpecification joinSpecification;
+
+    @XmlElement(name = "condition")
+    private ExpectedExpression condition;
+
+    @XmlElement(name = "columns")
+    private final List<ExpectedColumn> columns = new LinkedList<>();
```

Comment:
@lwtdev Maybe it is better to rename `columns` to `using-columns`.

Reply:
@jingshanglu OK, I will modify it. What about `condition`, do we should rename it to `on-condition`?

Reply:
@lwtdev I think both will do.